Dynatrace/dynatrace-operator
# Use version in application monitoring without CSI

# Description

In case of applicationMonitoring the `version` field only worked with the CSI driver but didn't work in standalone mode.

## How can this be tested?
Set the version field in an `applicationMonitoring` dynakube and check if the deployed sample apps use the correct version

## Checklist
